src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPMappingService.kt

```diff
@@ -45,6 +47,16 @@ class CSIPMappingService(@Qualifier("mappingApiWebClient") webClient: WebClient)
       .retrieve()
       .awaitBodyOrNullWhenNotFound()
 
+  suspend fun getDpsCsipMappings(nomisCSIPReportIds: List<Long>): List<CSIPReportMappingDto> =
+    webClient.get().uri {
+      it.path("/mapping/csip/nomis-csip-id")
+        .queryParam("nomisCSIPId", nomisCSIPReportIds)
```

Comment:
I never knew this worked - I always imagined you had to some how call .queryParam for each item - nice !